dist/js/script.js

```diff
@@ -2,38 +2,49 @@ $(document).ready(function(){
 
     //close promo text
     $('.promo__close').on('click', function () {
-        $('.promo_hide').hide('slow');
+        $('.promo_js').hide('slow');
```

Comment:
rename class to `js_promo` as you have it elsewhere

Reply:
but now you have a mix of classes. if you want to use `js` prefixes use it everywhere. so you will have:
`js-promo-close` class for calling action...